oak-security-spi/src/test/java/org/apache/jackrabbit/oak/spi/security/principal/TestPrincipalProvider.java

```diff
@@ -17,6 +17,8 @@
 package org.apache.jackrabbit.oak.spi.security.principal;
 
 import java.security.Principal;
+import java.util.ArrayList;
```

Comment:
https://sonarcloud.io/project/issues?id=org.apache.jackrabbit%3Ajackrabbit-oak&pullRequest=1914&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true

Reply:
thanks, fixed in https://github.com/apache/jackrabbit-oak/pull/1914/commits/47073558a422e1b46f1e6a47f101e2e90f20143d